tests/Tests/600__ai_apps/600__anaconda_commercial_edition/600__anaconda_commercial_edition.robot

```diff
@@ -110,3 +107,37 @@ Verify Anaconda In Kfdef
         END
     END
     Run Keyword And Continue On Failure     Should Contain  ${applications_names}  anaconda-ce
+
+Install Numpy Package Should Fail
+    [Documentation]     Tries to install python package "numpy" and checks
+    ...                 if the output matches the expected error message.
+    ...                 The installation should fail because runs before token validation
+    ${pkg_install_out}=     Run Cell And Get Output    !conda install -y numpy
+    ${pkg_install_out}=     Remove String Using Regexp    ${pkg_install_out}    ${\n}|${SPACE}
+    ${PKG_INSTALL_ERROR_MSG}=     Remove String Using Regexp    ${PKG_INSTALL_ERROR_MSG}    ${\n}|${SPACE}
+    Should Be Equal As Strings      ${PKG_INSTALL_ERROR_MSG}    ${pkg_install_out}
+
+Validate Anaconda Token
+    [Documentation]     Sets the token using "conda" command. It is necessary to start
+    ...                 using the conda  packages
+    Run Cell And Check Output    !conda token set ${ANACONDA_CE.ACTIVATION_KEY}    ${TOKEN_VAL_SUCCESS_MSG}
+    Capture Page Screenshot  anaconda_token_val_cell.png
+
+Install Numpy Package Should Be Successful
+    [Documentation]     Tries to install python package "numpy" and checks
+    ...                 if it runs without errors.
+    ...                 The installation should be successful because runs after token validation
+    Add And Run JupyterLab Code Cell In Active Notebook  !conda install -y numpy
+    Wait Until JupyterLab Code Cell Is Not Active
+    Run Cell And Check For Errors  import numpy as np
+    Capture Page Screenshot  conda_lib_install_result.png
+
+Check condarc File Content
+    [Documentation]     Checks the location and content of the "condarc" configuration file
+    ${condarc_cat_out}=     Run Cell And Get Output    !cat /etc/conda/condarc
+    ${condarc_cat_out}=     Remove String Using Regexp    ${condarc_cat_out}    ${\n}|${SPACE}
+    ${CONDARC_CAT_RESULT}=     Remove String Using Regexp    ${CONDARC_CAT_RESULT}    ${\n}|${SPACE}
+    Should Be Equal As Strings      ${CONDARC_CAT_RESULT}    ${condarc_cat_out}
+    Run Keyword And Continue On Failure
+    ...             Run Cell And Check Output   !cat $HOME/.condarc    cat: /opt/app-root/src/.condarc: No such file or directory
```

Comment:
Why are you expecting this file to not be present?
Would it be a bug if it were present?

Reply:
> Would it be a bug if it were present?


yes. 


> Why are you expecting this file to not be present?


because we expect `$HOME/.condarc` to be replaced by `/etc/conda/condarc`

Reply:
sounds good, resolving